index.html

```diff
@@ -63,7 +67,8 @@ <h3>Jamming to "Fields of Gold" by Sting</h3>
 As we walk in fields of gold
     </p>
     
-    <p>So she took her love<br>
+    <!--background for only one paragraf-->
+    <p  style="background:rgb(102, 151, 28);"> So she took her love<br>
```

Comment:
Please format the document. You can use Visual Studio auto-formatting by pressing ctrl+shift+i